blob/service.go

```diff
@@ -55,10 +57,18 @@ func NewService(
 // Allows sending multiple Blobs atomically synchronously.
 // Uses default wallet registered on the Node.
 // Handles gas estimation and fee calculation.
-func (s *Service) Submit(ctx context.Context, blobs []*Blob) (uint64, error) {
+func (s *Service) Submit(ctx context.Context, blobs []*Blob, options ...Option) (uint64, error) {
```

Comment:
We cannot do variadic params on the RPC :( I would suggest adding a new endpoint, it is less breaking and keeps the Submit signature clean for most people's use case

Reply:
i think there was a suggestion/decision to add `SubmitWithOptions` this morning?

Reply:
There could be lots of potential options, so I'd suggest to call it SubmitWithFee for this particular case